api/v1beta1/auth_config_types.go

```diff
@@ -175,6 +212,9 @@ type Metadata struct {
 	// +kubebuilder:default:=0
 	Priority int `json:"priority,omitempty"`
 
+	// Conditions that must match for Authorino to enforce this metadata config; otherwise, the config will be skipped.
+	Conditions []JSONPattern `json:"conditions,omitempty"`
```

Comment:
If there are no conditions set, is the default not to skip?

Reply:
Correct. No conditions means always enforce it.

Reply:
Ouch. Got it now. Description is wrong. On it!

Reply:
Perhaps add to the description:

```go
// If omitted, the config will be enforced every time.
```

?

Reply:
Yeah sounds good, it is a minor thing , it just caught my attention as I read it